docs/how-to-guides/rook-ceph-storage.md

```diff
@@ -0,0 +1,293 @@
+# Storage with Rook Ceph on Packet cloud
```

Comment:
I'm missing a bit what kind of storage we talk about here? Is it the storage providing PVs to the cluster?

Reply:
Yes storage that provides PVs, but then what other storage we might refer in the context of Kubernetes?

Reply:
From ceph perspective, it might also be Rados object storage (S3 alternative), hence the question. I'll think how we could rephrase existing title.

Reply:
Hm, some suggestions:
- Kubernetes PVs with Rook Ceph on Packet cloud
- Kubernetes storage with Rook Ceph on Packet cloud
- Rook Ceph CSI on Packet cloud

Reply:
Cool adding second option. :-)